FewCore/Component.swift

```diff
@@ -234,15 +234,19 @@ public class Component<S>: Element {
 	}
 
 	final private func enqueueRender() {
-		if renderQueued { return }
-
-		renderQueued = true
-
-		let observer = CFRunLoopObserverCreateWithHandler(kCFAllocatorDefault, CFRunLoopActivity.Exit.rawValue, 0, 0) { _, _ in
-			self.renderQueued = false
-			self.render()
-		}
-		CFRunLoopAddObserver(CFRunLoopGetMain(), observer, kCFRunLoopCommonModes)
+#if os(iOS)
```

Comment:
Kinda different. We want to coalesce all updates within the same runloop. But this is fine for now. I'll take a look at why the enqueueing wasn't working.